spi_nand_flash/src/nand.h

```diff
@@ -31,6 +47,7 @@ struct spi_nand_flash_device_t {
     uint32_t read_page_delay_us;
     uint32_t erase_block_delay_us;
     uint32_t program_page_delay_us;
+    ecc_data_t ecc_data;
```

Comment:
Perhaps a more descriptive name, such as "last_ecc_data", could be used?

Reply:
Since it contains both configuration settings and status, I named it ecc_data. However, can it be ecc_context, ecc_info, ecc_control, ecc_conf_stat?